V/Valhalla/build_tarballs.jl

```diff
@@ -0,0 +1,98 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Valhalla"
+version = v"3.3.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/valhalla/valhalla.git", "ea7d44af37c47fcf0cb186e7ba0f9f77e96f202a"),
+    ArchiveSource("https://github.com/phracker/MacOSX-SDKs/releases/download/10.15/MacOSX10.15.sdk.tar.xz", "2408d07df7f324d3beea818585a6d990ba99587c218a3969f924dfcc4de93b62"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/valhalla/
+
+# Handle Mac SDK <10.12 errors
```

Comment:
Why do you need to download the 10.15 sdk if we have already the 10.12?

Reply:
Yup, great point. Just need an env var here.